### PR TITLE
add javascript-astar@0.4.1 w/ git auto-update

### LINKED ISSUE #9540: [Request] Add astar
**Library name:** astar
**Git repository url:** https://github.com/bgrins/javascript-astar
https://www.npmjs.com/package/javascript-astar

### PR DESCRIPTION
**Pull request by lcd78706**
cc #9414, close #9540, cc @bgrins

Pull request for issue: #9540
Related issue(s): #9414 

@kennynaoh  Please review this PR. Thanks!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bgrins/javascript-astar
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/javascript-astar
 * License and its reference:  MIT  https://github.com/bgrins/javascript-astar/commit/a7f1698803a51da3526b441cb099651a5148598f#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R13
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 604
   - Count of watchers: 43
   - Count of forks: 149
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 15
   - Downloads in the last month: 87

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/javascript-astar/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "license": "MIT",
+  "homepage": "https://bgrins.github.io/javascript-astar",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bgrins/javascript-astar.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "astar.js"
```
Comment (kennynaoh) [inline ajax/libs/javascript-astar/package.json]: @lcd78706 I think we can use more flexible rule here

**Comment by kennynaoh** [comment]
LGTM, ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
Please fill homepage field in comment next time as you have it in package.json.

**Pull request merged**